### DISCUSSION TITLE
Why boost-coroutine is not supported on arm?

### DISCUSSION THREAD

**Discussion by ybogo** [microsoft/vcpkg#26947] [Q&A]
I am trying to build my project on macos m1 and getting an error
```
-- Bootstrapping vcpkg before install
-- Bootstrapping vcpkg before install - done
-- Running vcpkg install
Error: boost-coroutine:arm64-osx@1.78.0 is only supported on '!arm & !uwp & !emscripten'

-- Running vcpkg install - failed
```

But I built boost from source via b2 and got no problems. Maybe this constraint was added for previous versions of boost?

**Comment by autoantwort** [comment]
You can use `--allow-unsupported`

**Reply by ybogo** [reply]
Thanks!
Note: in cmake you can specify  -DVCPKG_INSTALL_OPTIONS=--allow-unsupported

**Reply by Osyotr** [reply]
Consider opening a PR for this
